manifests/s/SnapNet/SnapNet/0.28.0.0/SnapNet.SnapNet.locale.en-US.yaml

```diff
@@ -1,12 +1,29 @@
-# Created using wingetcreate 1.6.1.0
+# Created with YamlCreate.ps1 v2.4.1 $debug=AUSU.CRLF.5-1-26257-5000.Win32NT
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
 
 PackageIdentifier: SnapNet.SnapNet
 PackageVersion: 0.28.0.0
 PackageLocale: en-US
-Publisher: SnapNet
-PackageName: SnapNet
+Publisher: Snap.Net
+# PublisherUrl: https://github.com/stijnvdb88
+# PublisherSupportUrl: https://github.com/stijnvdb88/Snap.Net/issues

```

Comment:
Curious why you commented these out

Reply:
More that I forgot to uncomment them after adding them